02.1_Solar_System.rb

```diff
@@ -0,0 +1,172 @@
+### Solar System
+require "awesome_print"
+
+## Setup Classes
+class SolarSystem
+  AGE = 4_600
+  attr_reader :planets
+
+  def initialize(planets_array)
+    @planets = planets_array
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def planet_list
+    display = Array.new
+
+    @planets.each_index do |i|
+      display << "#{i + 1}. #{planets[i].name}\n"
+    end
+
+    return display
+  end
+
+end # End of Class SolarSystem
+
+class Planet
+  EARTH_YEAR = 365.2564
+
+  attr_accessor :user_name, :user_days, :user_age, :user_distance, :user_mass, :user_moons, :user_life
+  attr_reader :name, :earth_days, :earth_age, :distance_from_the_sun, :earth_mass, :has_moons, :known_life, :rel_age
+
+  def initialize(name, year, age, distance, mass, moons, life)
+    @name = name
+    @earth_days = year
+    @earth_age = age
+    @distance_from_the_sun = distance
+    @earth_mass = mass
+    @has_moons = moons
+    @known_life = life
+  end
+
+  def rel_age
+    return (SolarSystem::AGE - @earth_age.to_i) * Planet::EARTH_YEAR / @earth_days.to_f
```

Comment:
small bug: input to `@earth_days` doesn't check if this is a number or not. If it's not a number, `.to_f` makes it `0`, which divides by zero.